src/assets/styles/global-variables.scss

```diff
@@ -23,6 +23,7 @@ $accent-3-halftone: rgba(255, 0, 72, 0.5);
 $accent-4: rgba(46, 168, 122, 1);
 $accent-4-halftone: rgba(46, 168, 122, 0.5);
 $accent-4-transparent: rgba(46, 168, 122, 0);
+$gray-label: #506176;
```

Comment:
Instead of adding new color, let's just use an existing grey one, for example `$gray-1`